src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.ts

```diff
@@ -22,4 +22,18 @@ export class DashboardHelpComponent implements OnInit {
       subs.unsubscribe();
     });
   }
+
+  goToApiDocs() {
+    let tokenInput = null;
+    if (this.docsFormElement.nativeElement.children.length === 0) {
```

Comment:
I think this can be simplified if you just declare the input field in the html and then you only need to update it's value in the method.